vocabularies/Offline.xml

```diff
@@ -52,7 +52,7 @@ Please refer to the [Offline OData product home](https://help.sap.com/doc/f53c64
         </Annotation>
       </Term>
 
-      <Term Name="LocalDraft" Type="Core.Tag" DefaultValue="true" Nullable="false" AppliesTo="EntitySet">
+      <Term Name="LocalDraft" Type="Edm.String" Nullable="true" AppliesTo="EntitySet">
```

Comment:
```suggestion
      <Term Name="LocalDraft" Type="Edm.String" Nullable="false" AppliesTo="EntitySet">
```
What would `"@Offline.LocalDraft": null` mean? Isn't it sufficient to omit the annotation if there is no local draft?

Reply:
If there is no local-draft, then the annotation should of course be removed.
Regarding null value or default value, here I quote Evan's reply from the conversation above:
"the implicit default of null can then indicate that the draft set is linked with the only other entity set using the same entity type."

Reply:
Actually this is brittle: adding another entity set with the same type is a compatible change, but it would break the annotation.

It also puts burden on the client: look for the only other set with the same type, instead of just writing down what the author of the annotation already knows: the name of that other entity set.